react-native/README.md

```diff
@@ -42,6 +45,12 @@ npm run watch
 
 #### iOS
 
+Install cocoapods:
+
+```sh
+sudo gem install cocoapods --pre -v 1.0.0.beta.6
```

Comment:
if you use rbenv you don't have to sudo but i guess most people probably use the default os x ruby install so...